irohad/ametsuchi/impl/storage_impl.hpp

```diff
@@ -99,7 +99,7 @@ namespace iroha {
       const PostgresOptions postgres_options_;
 
      private:
-      std::unique_ptr<FlatFile> block_store_;
+      std::unique_ptr<FlatFileImpl> block_store_;
```

Comment:
Maybe use `FlatFile` here so that it is possible to use mock in the future? Because now it is not possible due to static methods hardcoded with `FlatFileImpl`.